src/App/Webhook/PdkWebhookManager.php

```diff
@@ -57,7 +57,13 @@ public function call($input, string $context = self::CONTEXT_WEBHOOK): Response
             return $response;
         }
 
-        $this->cronService->dispatch([$this, 'processWebhook'], $input);
+        /**
+         * TODO: restore async calling using cronService when this is fixed in WordPress
+         * besides fixing the wp_schedule_single_event, you also need a Facade to call processWebhook statically
+         * $this->cronService->dispatch([$this, 'processWebhook'], $input);
+         * For now: process immediately
+         */
+        $this->processWebhook($input);
```

Comment:
maar moet dit hier? dit kan toch gewoon in wordpress opgelost worden door dispatch dit direct te laten doen?

Reply:
Ok ik sluit de pr, ik deed het hier om 100% zeker te zijn dat het in alle platformen zou werken, en dat ik niet alleen een fix voor WordPress zou maken.